.github/workflows/tests.yml

```diff
@@ -63,6 +63,9 @@ jobs:
       - name: Build
         run: cargo build --verbose
 
+      - name: Cargo fmt
+      - run: cargo fmt
```

Comment:
Pretty sure you want `cargo fmt --check` for the CI, otherwise it will just apply formatting without failing.

Reply:
indeed, thanks for the catch!